gooey/languages/german.json

```diff
@@ -1,37 +1,50 @@
 {
   "browse": "Durchsuchen",
   "cancel": "Abbrechen",
+  "checkbox_label": "Aktiviere",
+  "choose_colour": "",

```

Comment:
Here the translation is missing. What do you think of "Farbe auswählen"

Reply:
That works for me 😀